components/filter/test/filter-dimension-set-value.test.js

```diff
@@ -14,16 +14,47 @@ describe('d2l-filter-dimension-set-value', () => {
 	});
 
 	describe('data change', () => {
-		it('fires data change event when its data changes', async() => {
-			const elem = await fixture(valuefixture);
-			const eventSpy = spy(elem, 'dispatchEvent');
-			elem.text = 'Test';
-
-			const e = await oneEvent(elem, 'd2l-filter-dimension-set-value-data-change');
-			expect(e.detail.valueKey).to.equal('value');
-			expect(e.detail.changes.size).to.equal(1);
-			expect(e.detail.changes.get('text')).to.equal('Test');
-			expect(eventSpy).to.be.calledOnce;
+		[
+			{ property: 'count', value: 1 },
+			{ property: 'disabled', value: true },
+			{ property: 'selected', value: true },
+			{ property: 'text', value: 'Test' }
+		].forEach(condition => {
+			it(`fires data change event when its ${condition.property} data changes`, async() => {
+				const elem = await fixture(valuefixture);
+				const eventSpy = spy(elem, 'dispatchEvent');
+
+				if (condition.property === 'count') elem.count = condition.value;
+				else if (condition.property === 'disabled') elem.disabled = condition.value;
+				else if (condition.property === 'selected') elem.selected = condition.value;
+				else if (condition.property === 'text') elem.text = condition.value;
```

Comment:
These can be simplified as `elem[condition.property] = condition.value`.

Reply:
Made the change.